### ISSUE TITLE
[bug] Fibaro motion sensor: Zwave JS reports no motion after exactly 5 minutes even if motion is still there

### ISSUE THREAD

**Issue by wauswaus** [zwave-js/zwavejs2mqtt#990]
### Version

- [x] I am using HomeAssistant. **Or:** a developer has told me to come here.
_I do run on HomeAssistant, however this issue not HA related._
- [ ] I have checked the troubleshooting section and my problem is **not** described there.
- [ ] I have read the changelog problem was **not** mentioned there.

Build/Run method

- [x] Docker (home assistant add-on)
- [ ] PKG
- [ ] Snap package
- [ ] Manually built (git clone - npm install - npm run build )

zwavejs2mqtt: 3.0.3
zwave-js: 7.0.1
home assistant: core-2021.3.4

### Describe the bug
I migrated from OZW 1.6 to Zwave JS. I am running it in Home assistant, but I do not use MQTT. The home assistant integration connects to the websocket.

Two of the sensors I use is a Fibaro FGMS001 motion sensor. When running OZW, no issues.

When the sensor detects motion, it flashes and a motion report is received by Zwavejs. The sensor only flashes again after it stopped seeing motion for the setup duration amount of time and afterwards it sees motion again.
If I keep moving in front of the sensor, it flashes and reports motion. For some reason, after exactly 5min (almost to the ms accurate) I see in the logging that the motion value goes from detected to not detect. Weird thing is that this happens while I keep moving in front of the sensor. When it happens, the sensor does not flash again so it never stopped seeing motion. When I then stop moving, I see again in the logging the motion value changed from no motion to no motion. When I then move again, sensor flashes and in logging I see the motion detected.

In the attached zwavejs logging, I trigger the sensor at 16:09:18 (18:09:18 in zwavejs2mqtt logging) and I keep moving in front of it.
At 16:14:18 the sensor status goes from motion detected to not detected. However, it appears there is no sensor message that triggered this.
I leave the room and at 16:15:28 the sensor actually reports no motion.

It almost looks like there is a maximum somewhere defined that motion is allowed to be detected, however I do not see such a parameter in the configuration settings of the sensor.

I have this behaviour on both Fibaro motion sensors. When I test for such behaviour on a Shenzen Neo PD03Z motion sensor, I do not see this issue.

### To Reproduce

Start moving in front of the (Fibaro FGMS001) and keep monitoring the motion value changed event.

### Expected behavior

Motion is still detected. No motion detected only when there is really no motion and sensor reports no motion.

### Additional context

See attached logging (zip)
[logging.zip](https://github.com/zwave-js/zwavejs2mqtt/files/6245283/logging.zip)
.

**Comment by blhoward2** [comment]
Some motion sensor devices have a parameter that resets the device after a period of time. Have you checked the parameters?

**Comment by wauswaus** [comment]
I did check the parameters and did not find any as stated in my opening post. [Link](https://devices.zwave-js.io/?jumpTo=0x010f:0x0801:0x1002:3.3) to device database.
When migrating from OZW to zwavejs I did not touch the sensor parameters and in OZW I did not have this behaviour.

**Comment by robertsLando** [comment]
@AlCalzone

**Comment by AlCalzone** [comment]
Exactly 5 minutes is the time after which zwave-js auto-idles notification variables on older devices. This is recommended by the Z-Wave specs and is necessary for many devices that do not reset their notifications and seem to get stuck in an active state.

Interestingly, after 6 minutes, the device itself sends an idle notification. Is that something configurable (i.e. how long the motion sensor stays "active")? If not, I see two options:
* Add a "good" compat flag to disable the auto-idling for devices that send them on their own
* Add a heuristic to turn off auto-idling if the device sends an idle notification at some point. This may probably result in a false reset once before the heuristic comes into effect.

**Comment by wauswaus** [comment]
The message seen after 6 minutes is because I stopped dancing in front of the sensor and left the room to gather the logs :)
To answer your question, no, I can't configure any time to get the sensor to resent the motion notification. Here I am, thinking Fibaro as a big brand, designed their products well and adhered to the standard...

What time scale are we talking of the get a 'fix' in place for this? After 5 minutes my lights now always switch off. With summer coming in Europa it is currently not that big of a deal, but else I have to start thinking of reverting back to OZW for the time being. The misses is patient till a certain extend with my hobby. 

Is there anything else I can do to circumvent this from happening? Poll the sensor every minute when motion is detected?

**Comment by AlCalzone** [comment]
To be clear, I don't think fibaro is doing anything wrong here. The auto reset is in place because the requirement for the idle message was only added in a later version of the Notification CC.
Maybe it is worth swapping the behavior to be opt in. This will require code changes so there's probably nothing you can do right now. Polling won't help because of how notifications work.

**Comment by ettiennegous** [comment]
Might be related but after a recent update to the latest zwave Js my Aoetec Multisensors stopped reporting motion too, the lux and humidity reporting appears unaffected but motion no longer fires on the two I have

**Comment by AlCalzone** [comment]
My multi sensor 6 works just fine 🤷🏻‍♂️

**Comment by wauswaus** [comment]
I checked and unfortunately the sensor doesn't have a binary entity I could use.
Opt in for the auto idling sounds good. How should a user be able to opt in? I guess on a per node base?

You want me to make an issue for this on the zwave js server Git or you need anything else from me to get this resolved?

**Comment by AlCalzone** [comment]
This would be an option in the device config files.

**Comment by Hooverjc** [comment]
Just wanted to report that I also switched from OZW to zwaveJS2mqtt yesterday and am having the same exact issue with all 5 of my GE/Jasco 34193/ZW6302 motion sensors. They seem to work well for a few minutes but with any sustained motion, regardless of the timeout I set on the device configuration, they go "Idle" and basically become useless until the motion has stopped long enough for the device to reset itself which is obviously impractical. I rolled back to OZW and the issue is not present. Logging this comment for visibility/awareness but also in hopes that it lets me know once this is resolved so I can confirm this works for me too. Thansk!
